fec/fec/static/js/calendar.js

```diff
@@ -339,6 +348,27 @@ Calendar.prototype.handleEventClick = function(calEvent, jsEvent, view) {
   }
 };
 
+Calendar.prototype.handleKeypress = function(e) {
+  if (e.keyCode === 13) {
+    $(e.target).click();
+  }
+};
+
+Calendar.prototype.controlPopoverFocus = function(e) {
+  var $target = $(e.target);
+  var $popover = $('.fc-popover');
```

Comment:
Is there only one `.fc-popover` at a time? Can we scope this selector to the current calendar?